lib/safer_rails_console/patches/boot/sandbox_flag.rb

```diff
@@ -76,6 +76,13 @@ def parse_arguments(arguments)
 elsif SaferRailsConsole::RailsVersion.five_zero?
   require 'rails/commands/commands_tasks'
   ::Rails::CommandsTasks.prepend(SaferRailsConsole::Patches::Boot::SandboxFlag::Rails::CommandsTasks50)
+elsif SaferRailsConsole::RailsVersion.five_one?
+  require 'rails/command'
+  require 'rails/commands/console/console_command'
+  ::Rails::Command::ConsoleCommand.class_eval do
```

Comment:
We want it to be `nil` to know if the user explicitly specifies `--sandbox` (`true`), `--no-sandbox` (`false`), or does not not pass a flag at all (`nil`).  We use this to enforce automatically specified rules (auto-sandbox).  I'll add a comment explaining this.